src/openzeppelin/account/library.cairo

```diff
@@ -164,6 +165,13 @@ namespace Account {
     ) {
         alloc_locals;
 
+        // assert version != 0
```

Comment:
If we are asserting version 1, shouldn't we update the preset Account contracts adding the __validate__ method to be compatible? I haven't found the signature of __execute__ and __validate__ functions documented  in new version though.

Reply:
Resolved offline